lutris/sysoptions.py

```diff
@@ -205,6 +205,14 @@ def get_dri_prime():
         'advanced': True,
         'help': "Disables process monitoring of games, Lutris won't detect when game quits."
     },
+    {
+        'option': 'disable_compositor',
+        'label': "Disable desktop effects",
+        'type': 'bool',
+        'default': True,
```

Comment:
Default to False, this is the current behavior.

Reply:
I've set it to False, though I'm not sure why it shouldn't be enabled by default.